.github/workflows/cli.yaml

```diff
@@ -48,22 +38,14 @@ jobs:
             - name: Install dependencies
               run: pnpm install
 
-            - name: Test / Coverage
-              working-directory: ./apps/cli
-              run: pnpm run test:coverage
+            - name: Build
+              run: pnpm build --filter @cartesi/cli
+
+            - name: Test
+              run: pnpm test --filter @cartesi/cli
 
             - name: "Report Coverage"
               if: always()
               uses: davelosert/vitest-coverage-report-action@v2
               with:
                   working-directory: ./apps/cli
```

Comment:
Like I commented before, does running with filter will create the coverage/ directory at the root or inside apps/cli ?

Reply:
inside the cli. turbo basically runs tests recursively in all projects